src/plot_api/plot_api.js

```diff
@@ -1753,13 +1765,16 @@ function _relayout(gd, aobj) {
         fullLayout = gd._fullLayout,
         keys = Object.keys(aobj),
         axes = Plotly.Axes.list(gd),
-        i;
+        arrayEdits = {},
+        arrayStr,
+        i,
+        j;
```

Comment:
FYI: I think by far the biggest manual change required to move from eslint to standard is that variable declarations like this must manually be split. The general rule is that initialized declarations must each have their own `var` and uninitialized declarations are fine in the same statement. So no particular feedback here, just that every time I see these, I worry about the big cleanup pass ahead 😬